utils/putDataToIpdc.js

```diff
@@ -0,0 +1,39 @@
+import * as jsonld from 'jsonld';
+import { IPDC_JSON_ENDPOINT, IPDC_X_API_KEY } from '../env-config';
+export async function putDataToIpdc(subjectsAndData){  
+  //try catch so it doesnt messup other parts of the cron job
+  try {
```

Comment:
I would for now let it crash, so i gets picked up in another attempt. It is not a big deal if it gets published twice somewhere else for now.